CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.48.4] - 2025-10-09
```

Comment:
Technically you don't need to change the version in this PR -- you can save it for when we merge develop to master. But it's fine doing it here if you want. That said, this should be a new minor version (1.49.0)